### PR TITLE
stability/verify unification

### LINKED ISSUE #9874: Move stability checking to `wpt run --verify`
Since we developed the stability checker, wptrunner got a `--verify` flag for checking test stability. Ideally instead of having separate codepaths here we could keep all the travis handling logic from the check_stability script but do the actual run using the `--verify` option to wpt run, so that we are confident that the same CI checks apply to both.

### LINKED ISSUE #10921: Make --stability just use the --verify code
To roughly quote @jgraham:

> so `--stability` is almost exactly run on Travis, `--verify` is almost exactly what’s run on Mozilla infra; `--verify` is more flexible (with/without restarts, with/without chaos mode, varying number of iterations); `--stability` just does 10 iters with restarts

So basically `--stability` is a subset of the behaviour and should probably cease being a separate codepath.

Note that `--stability` is currently a feature of `wpt run` and *not* `wptrunner`.

We need to make sure the output doesn't change when we move this over.

### PR DESCRIPTION
**Pull request by gsnedders**
Fixes #10921, and will fix #9874.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/10988)

### CODE REVIEW

**Comment by gsnedders** [comment]
Seems like I posted https://github.com/w3c/web-platform-tests/issues/9874#issuecomment-388892276 in the wrong place:

> So what I haven't done so far is move over to running the multi-part verify (with/without restarts, with/without chaos mode in Firefox), but it does now use the verify code.
>
> At the moment, it's still calling into the code through Python, rather than letting it run separately and then reading the logs. If we want to avoid going through the logs, we'd need to either make `check_stability` (in tools/wptrunner/wptrunner/stability.py) return something useful or call get_steps ourselves.

File: tools/wptrunner/wptrunner/wptcommandline.py
Comment (jgraham) [inline tools/wptrunner/wptrunner/wptcommandline.py]: You can use `help = argparse.SUPPRESS` to remove this option from the help.

File: tools/wptrunner/wptrunner/wptrunner.py
Comment (jgraham) [inline tools/wptrunner/wptrunner/wptrunner.py]: Maybe log a warning here.

**Review by jgraham — CHANGES_REQUESTED** [review]
Just a few small changes to request.

Can you push an unstable test, and a stable one, to this PR so that we can see how the travis output looks?

**Comment by gsnedders** [comment]
@jgraham now done; sorry I missed your comments before!

**Comment by foolip** [comment]
> Can you push an unstable test, and a stable one, to this PR so that we can see how the travis output looks?

Ping @gsnedders

**Comment by gsnedders** [comment]
Um, yeah, seems I deleted them locally. Now readded.

**Comment by gsnedders** [comment]
(labelled because that commit needs to go before merging)

**Comment by foolip** [comment]
The Travis output looks about the same as before, so that seems to be working.

**Review by jgraham — APPROVED** [review]
Please remove the [DO NOT MERGE] commit :)

**Pull request merged**
